src/domain/session/room/timeline/tiles/GapTile.js

```diff
@@ -55,7 +70,18 @@ export class GapTile extends SimpleTile {
         let canFillMore;
         this._siblingChanged = false;
         do {
-            canFillMore = await this.fill();
```

Comment:
if you scroll while offline, this will retry the fill while the previous invocation of fill is still waiting for the connection to come back online. notifyVisible should probably exit early if we're waiting for the connection to come back online.

Reply:
@MidhunSureshR Could you add a word why your resolved this?

Reply:
While we wait for reconnection, `this._isLoading` is still true, so further calls to fill method will return early.